Libs/Widgets/ctkAxesWidget.cpp

```diff
@@ -257,8 +257,8 @@ void ctkAxesWidget::paintEvent(QPaintEvent *)
   for (int i = 0; i < 6; ++i)
   {
     //if (d->HighlightAxes)
```

Comment:
Could you remove the commented out code (and associated curly braces)?

Reply:
@lassoan  done.